recipes-core/initrdscripts/initramfs-framework/aosupdate

```diff
@@ -0,0 +1,262 @@
+#!/bin/sh
```

Comment:
Commit message: remove period at the end

Reply:
Ok, I will correct it.